spec/models/views/reports/public_submission_data_spec.rb

```diff
@@ -0,0 +1,43 @@
+# coding: utf-8
+require 'rails_helper'
+
+RSpec.describe Views::Reports::PublicSubmissionData do
+
+  subject(:data) { described_class.new }
+
+  describe '#submission_all_time_total' do
+    subject { data.submission_all_time_total }
+
+    it { is_expected.to be_a Integer }
```

Comment:
I suppose this is about importance (how important it is if the report wasn't working correctly). But I'd test some specific values, read from the DB. You can use factories to prime it with the specific setup and test your queries are working. Let's discuss it tomorrow, though.